src/main/java/me/arminb/sara/entities/BaseEntity.java

```diff
@@ -0,0 +1,33 @@
+package me.arminb.sara.entities;
+
+import java.util.Date;
+
+public class BaseEntity {
+    private String id;
```

Comment:
Change these fields to protected so they can be accessible from the child classes.